libvcx/src/api_lib/api_c/issuer_credential.rs

```diff
@@ -125,6 +126,8 @@ pub extern fn vcx_issuer_create_credential(command_handle: CommandHandle,
 /// #Returns
 /// Error code as a u32
 #[no_mangle]
+#[deprecated(since = "0.28.0", note = "Call methods 'vcx_issuer_build_credential_offer_msg' and \
```

Comment:
We should remove these deprecated functions soon, they only add clutter

Reply:
agreed!